client/src/actions/index.js

```diff
@@ -1,6 +1,6 @@
-export function loadContacts() {
+export function loadContacts(id) {
```

Comment:
I think there was some confusion on the client part of the project
You should not be changing any of the code that was already there. You will only be adding new code